apis/console/src/lib.rs

```diff
@@ -39,6 +39,9 @@ impl<S: Syscalls, C: Config> Console<S, C> {
     /// This is an alternative to `fmt::Write::write`
     /// because this can actually return an error code.
     pub fn write(s: &[u8]) -> Result<(), ErrorCode> {
+        if s.is_empty() {
```

Comment:
Add a comment explaining why this is required and post an issue to https://github.com/tock/tock